src/Loggable/LoggableListener.php

```diff
@@ -100,6 +101,8 @@ public function getSubscribedEvents()
     /**
      * Maps additional metadata
      *
+     * @param LoadClassMetadataEventArgs $eventArgs
```

Comment:
I think we should not support this type here, as it is breaking the contravariance.

Reply:
What do you mean?

The thing here is that is actually receiving a `LoadClassMetadataEventArgs` instance, since we are still supporting PHP 7.2 (with partial support of contravariance) we cannot change it in the method signature.

Reply:
I mean this situation is not allowed even in PHP 7.2, where a warning is emitted if we could replace `EventArgs` by `LoadClassMetadataEventArgs`. In PHP 8, it results in a fatal error: https://3v4l.org/bvZdv.
I think we should work in the method's body instead, avoiding calls for methods not defined in `EventArgs` or adding a call to `assert($eventArgs instanceof LoadClassMetadataEventArgs)`.
IMO, those are the only ways we have to avoid breaking the contravariance in a safer way, without bringing a wrong message through the docblocks: https://phpstan.org/r/5fa3b930-245b-461e-9077-a38a625c9b6b.

Reply:
I think here the case is different, in these classes the `loadClassMetadata` method is not inherited. It is more this case: https://3v4l.org/E3vU8.

Anyway, I could add `assert($eventArgs instanceof LoadClassMetadataEventArgs)`.

Reply:
Forget about the comment then. I thought these methods were overriding this one:
https://github.com/doctrine-extensions/DoctrineExtensions/blob/bcc142745aa97a77d58af591a72d4341b2615ac7/src/AbstractTrackingListener.php#L53-L57

but checking the ancestors, I see there is no inheritance for this.
Thank you for the clarification.